### PR TITLE
Algebraic_kernel_d: Remove RS

### LINKED ISSUE #7771: Remove RS/RS3 from CGAL (was Non existing link to RS / RS3)
In Documentation/doc/Documentation/Third_party.txt(Manual/thirdparty.html) there is a link to:
```
http://vegas.loria.fr/rs/
```
but this link does not exist anymore.

### PR DESCRIPTION
**Pull request by afabri**
## Summary of Changes

I removed RS in the third party library page, models in the documentation that are wrappers, wrappers, examples, and tests. 
I did not go through the depreciation process, as I do not expect anybody to use it, as RS has no official support.

Todo: Add to the change log

## Release Management

* Affected package(s): Algebraic_kernel_d
* Issue(s) solved (if any): fix #7771

### CODE REVIEW

File: Algebraic_kernel_d/doc/Algebraic_kernel_d/Algebraic_kernel_d.txt [hunk 1]
```diff
@@ -376,12 +341,6 @@ Michael Hemmer and Michael Kerber, respectively. Notwithstanding, the authors al
 contribution of all authors of the \exacus project,
 particularly the contribution of Arno Eigenwillig, Sebastian Limbach and Pavel Emeliyanenko.
 
-The two univariate kernels that interface the library RS \cgalCite{cgal:r-rs} were
```
Comment (afabri) [inline Algebraic_kernel_d/doc/Algebraic_kernel_d/Algebraic_kernel_d.txt:379]: I do not rewrite history like Stalin removing Trotsky from group photos. What would be a good text to say that during some time there was support for RS?  @pougetma any suggestion?

Comment (pougetma) [reply Algebraic_kernel_d/doc/Algebraic_kernel_d/Algebraic_kernel_d.txt:379]: I would propose to just slightly modify the text and add a bibref: 
In 2010, two univariate kernels that interface the library RS \cgalCite{cgal:r-rs} were
written by Luis Pe&ntilde;aranda and Sylvain Lazard \cgalCite{cgal:2009-ESA}.
Both models interface the library RS \cgalCite{cgal:r-rs} by Fabrice Rouillier.
The authors want to thank Fabrice Rouillier and Elias Tsigaridas for
strong support and many useful discussions that lead to the integration of RS.
Due to lack of maintenance, these kernels have been removed in 2024.

@inproceedings{cgal:2009-ESA,
author = {Lazard, Sylvain and Pe\~{n}aranda, Luis and Tsigaridas, Elias},
title = {Univariate Algebraic Kernel and Application to Arrangements},
year = {2009},
isbn = {9783642020100},
publisher = {Springer-Verlag},
address = {Berlin, Heidelberg},
url = {https://doi.org/10.1007/978-3-642-02011-7_20},
doi = {10.1007/978-3-642-02011-7_20},
booktitle = {Proceedings of the 8th International Symposium on Experimental Algorithms},
pages = {209–220},
numpages = {12},
location = {Dortmund, Germany},
series = {SEA '09}
}

Comment (afabri) [reply Algebraic_kernel_d/doc/Algebraic_kernel_d/Algebraic_kernel_d.txt:379]: That sounds perfect although "lack of maintenance" is more the fact that RS is not really available.  @pougetma can you just change it, or do you want me to add it ?

Comment (afabri) [reply Algebraic_kernel_d/doc/Algebraic_kernel_d/Algebraic_kernel_d.txt:379]: I am on it.

Comment (afabri) [reply Algebraic_kernel_d/doc/Algebraic_kernel_d/Algebraic_kernel_d.txt:379]: See [here ](https://cgal.github.io/8400/v0/Algebraic_kernel_d/index.html#title18)

Comment (pougetma) [reply Algebraic_kernel_d/doc/Algebraic_kernel_d/Algebraic_kernel_d.txt:379]: Ok, let's change "Due to lack of maintenance," for "Due to lack of availability of RS," I'm fine if you commit the fix at the right place ;-) thanks!

**Comment by sloriot** [comment]
Successfully tested in CGAL-6.0.1-Ic-345

**Comment by sloriot** [comment]
Successfully tested in CGAL-6.1-Ic-3

**Pull request merged**